src/index_feature_list.cpp

```diff
@@ -31,11 +33,12 @@ IndexFeatureList::IndexFeatureList()
 }
 
 bool
-IndexFeatureList::CheckFeature(IndexFeature feature) {
-    if (static_cast<uint32_t>(feature) >= feature_count_) {
-        throw std::runtime_error("wrong feature");
+IndexFeatureList::CheckFeature(IndexFeature feature) const {
+    auto val = static_cast<uint32_t>(feature);
+    if (val >= feature_count_ or val == 0) {
+        throw std::invalid_argument(fmt::format("wrong feature value: {}", val));
     } else {
-        auto [x, y] = get_pos(static_cast<uint32_t>(feature));
+        auto [x, y] = get_pos(val);
```

Comment:
where is define for `get_pos` ?

Reply:
https://github.com/antgroup/vsag/blob/main/src/index_feature_list.cpp#L23